interactive/notifications.py

```diff
@@ -22,3 +25,28 @@ def notify_registration_request_submitted(full_name, job_title, organisation, em
     full_name_link = slack.link(email, full_name, is_email=True)
     message = f"{full_name_link} ({job_title}) from {organisation} has registered their interest in using OpenSAFELY Interactive"
     slack.post(text=message)
+
+
+def send_welcome_email(email, context):
+    subject = "Welcome to OpenSAFELY Interactive"
+    html_body = render_to_string("emails/welcome_email.html", context)
+    text_body = _convert_html(html_body)
+
+    msg = EmailMultiAlternatives(
+        subject=subject,
+        from_email="team@opensafely.org",
+        to=[email],
+        body=text_body,
+    )
+    msg.attach_alternative(html_body, "text/html")
+    msg.send()
+
+
+def _convert_html(raw_html):
+    text_maker = html2text.HTML2Text()
+    text_maker.ignore_images = True
+    text_maker.ignore_emphasis = True
+    text_maker.ignore_links = True
+    text_maker.body_width = 0
+    raw_text = text_maker.handle(raw_html)
+    return raw_text
```

Comment:
TIL about `html2text`!

Have a look at how `incuna-mail` is used [in job-server](https://github.com/opensafely-core/job-server/blob/07969b387193c43874a05ac087d5e542df51b4f4/jobserver/emails.py), you might find that approach interesting too.  It's a fairly simple library (that I had a hand in writing at a previous job), to provide a nicer interface to Django's email framework.  The ability to pass in HTML and text templates is what always brings me back to it.